deployments/ecmps/release-v2.0-fix/Sprint8/5995/camdecmpswks/copy_monitor_plan_qa_data_to_workspace.sql

```diff
@@ -0,0 +1,217 @@
+-- PROCEDURE: camdecmpswks.copy_monitor_plan_qa_data_to_workspace(text[])
+
+DROP PROCEDURE IF EXISTS camdecmpswks.copy_monitor_plan_qa_data_to_workspace(text[]);
```

Comment:
In this version, we need to make sure the old version of the procedure is dropped.